CMakeLists.txt

```diff
@@ -51,7 +51,9 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
 endif()
 
 # Install in project directory
-set(CMAKE_INSTALL_PREFIX ${PROJECT_SOURCE_DIR})
+if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+	set(CMAKE_INSTALL_PREFIX ${PROJECT_SOURCE_DIR})
+endif()
 # Extra CMake modules (Find*.cmake) are in CMakeModules/
```

Comment:
Currently the build doesn't work on Windows anyway, but even then an user can just set the prefix themselves if they want to.

Most of the time it will just be using cpack (`make package`) to generate zips or installers for Windows anyway.

Personally I'd prefer to have the variables set correctly